app/models/neighbour_response.rb

```diff
@@ -0,0 +1,174 @@
+# frozen_string_literal: true
+
+class NeighbourResponse
+  include ActiveModel::API
+  include ActiveModel::Attributes
+  include ActiveModel::Model
```

Comment:
ActiveModel::API and ActiveModel::Model are the same but the latter may gain features in the future so we use the former to ensure forward compatibility.